scripts/aws/create-kubernetes-cluster.go

```diff
@@ -227,7 +225,7 @@ func (ac *AWSCluster) createEksWorkerNodes(cfClient *cloudformation.CloudFormati
 	// with Kubernetes version 1.14.7
 	// for region us-east-2.
 	// Amazon EKS-Optimized AMI list: https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-ami.html
-	eksAmi := aws.String("ami-0b031080918049726")
```

Comment:
Could you please describe this change. It is not clear about this constant. Thanks.

Reply:
Just changed AWS ami image to more recent. Does not affect anything for us.
I found the way to autoselect aws ami image. Will apply in another pull request